docs/codeql/ql-language-reference/ql-language-specification.rst

```diff
@@ -276,7 +277,7 @@ Type references
 
 With the exception of class domain types and character types (which cannot be referenced explicitly in QL source), a reference to a type is written as the name of the type. In the case of database types, the name includes the at sign (``@``, U+0040).
 
-::
+.. code-block:: ql
 
    type ::= (moduleId "::")? classname | dbasetype | "boolean" | "date" | "float" | "int" | "string"
```

Comment:
The `?` symbol on line 282 (above) shows up in the preview with a red box round it. I have no idea why and whether or not this is a problem, but thought I should mention it.
![image](https://user-images.githubusercontent.com/1877141/99106511-3fe5f500-25dc-11eb-9d04-bd3e34aab7cf.png)

Reply:
This is also seen at other locations in this file. I'm suspicious that none of the code in this file should have QL syntax highlighting.

Reply:
You're right. I'll turn off the syntax highlighting for this file.